### ISSUE TITLE
`ValidationError` doesn't fail with exit code 1

### ISSUE THREAD

**Issue by tusharmath** [tailcallhq/tailcall#333]
### Prerequisites

- [x] I have checked the existing issues to ensure this is not a duplicate.
- [x] I am running the latest version of Tailcall.

### Describe the bug

TC server when pointed to an invalid file, throws an validation error but exits with status code 0.

### Steps to reproduce

1. Create the following file `test.graphql`

   ```graphql
   schema @schema {
     query: Query
   }

   type Query {
     users: [User] @http(path: "/users")
   }
   ```

2. Run the check command on the file

   ```
     tc check ./assets/jsonplaceholder.graphql
     Validation Error
       • No base URL defined [Query, users, @http]

     echo $?
     0 # should be 1
   ```

### Expected behavior

Fail with exit code Err.

### Actual behavior

Fails with exit code OK.

### Environment information:

- **Operating System**: All
- **Tailcall Version**: 0.9.0
- **GraphQL Client/version** NA

### Additional context

Learn more about exit codes here — https://rust-cli.github.io/book/in-depth/exit-code.html
\*\*

**Comment by ayewo** [comment]
@tusharmath what exit code would you like it to emit?

**Comment by tusharmath** [comment]
@ayewo we could use `78`. We could use the exit code crate for this also — https://docs.rs/exitcode/latest/src/exitcode/lib.rs.html#89

**Comment by ayewo** [comment]
@tusharmath 
I'm curious which style of working is your project adopting with Algora:
1. let the first contributor that uses the `/attempt` command to work on an issue; or
2. let anyone attempt an issue right away. Multiple contributors can use the `/attempt` command. 

The upside of style 2 is that you get multiple contributions on a single issue (as is the case here) but you only pay one person. The downside for contributors obviously is that if their work is not accepted they get nothing for their time.

FWIW, the Remotion project uses [style 1](https://github.com/remotion-dev/remotion/issues/2495#issuecomment-1683640507) which avoids the scenario described in style 2.

**Comment by tusharmath** [comment]
I would prefer one.

**Comment by kremedev** [comment]
Bounties at this price range and difficulty generally are style 2

**Comment by tusharmath** [comment]
Agree with @kremedev

**Comment by ayewo** [comment]
> Bounties at this price range and difficulty generally are style 2

Nope, this is not codified anywhere. The style of working is entirely up to each project to decide which is why I asked to confirm.

**Comment by kremedev** [comment]
I'm just telling what I have seen so far

**Comment by tusharmath** [comment]
fixed in #336
